dydx/dydxPresenters/dydxPresenters/_v4/GlobalWorkers/Workers/dydxTransferSubaccountWorker.swift

```diff
@@ -38,6 +39,25 @@ final class dydxTransferSubaccountWorker: BaseWorker {
                 }
             }
             .store(in: &subscriptions)
+
+        AbacusStateManager.shared.state.accountBalance(of: AbacusStateManager.shared.environment?.usdcTokenInfo?.denom)
```

Comment:
Let's combine this with the logic from line 24-41... since the source data is the same.